tests/test_mapping.rs

```diff
@@ -116,6 +117,16 @@ fn test_delitem() {
 #[test]
 fn mapping_is_not_sequence() {
     Python::with_gil(|py| {
+        // downcast to PyMapping requires isinstance(<cls>, collections.abc.Mapping) to pass, so we
+        // have to register the class first
+        PyModule::import(py, "collections.abc")
+            .unwrap()
+            .getattr("Mapping")
+            .unwrap()
+            .getattr("register")
+            .unwrap()
+            .call1((Mapping::type_object(py),))
+            .unwrap();
```

Comment:
I suppose in absence of #991 users may find it useful to have this exposed as a documented API to improve discoverability?! Not completely sure, what do you think?

Reply:
Yeah I was thinking something like `py.register_abc_subclass<Mapping>(ABC::Mapping).unwrap()` for example. Like you say it might only be useful until #991 is available but easier and more discoverable than the above.